<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ouvrage_Interface</class>
 <widget class="QDialog" name="Ouvrage_Interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ouvrage_Interface</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Auteurs:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="auteurs_ouvrage">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>481</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Titre:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="titre_ouvrage">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>481</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="annee_ouvrage">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>190</y>
     <width>81</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>2022</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ann√©e d'edition:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Code ISBN:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_ouvrage">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>481</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Editeur:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>380</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ville d'edition:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editeur_ouvrage">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>310</y>
     <width>481</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ville_ouvrage">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>370</y>
     <width>481</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>500</y>
     <width>176</width>
     <height>28</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Ouvrage_Interface</receiver>
   <slot>valideEnregistrementOuvrage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>513</y>
    </hint>
    <hint type="destinationlabel">
     <x>378</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Ouvrage_Interface</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>513</y>
    </hint>
    <hint type="destinationlabel">
     <x>378</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>valideEnregistrementOuvrage()</slot>
 </slots>
</ui>
